docs/README.md

```diff
@@ -1,58 +1,44 @@
-# VTEX React App Template
+# VTEX Product List Context app
```

Comment:
In the app's name, you can add only the name itself. Such as: 
# Product List Context